flutter-ory-network/README.md

```diff
@@ -20,6 +20,31 @@ Create .env file with your project url in the root folder of the Flutter app
 ORY_BASE_URL=https://{your-project-slug}.projects.oryapis.com
 ```
 
+### Google Sign In
+
+If you use Google Sign In on Android, add following variable to .env file
+
+```env
+WEB_CLIENT_ID={web-client-id}.apps.googleusercontent.com
+```
+
+If you use Google Sign In on iOS, add following variable to .env file
+
+```env
+
+IOS_CLIENT_ID={ios-client-id}.apps.googleusercontent.com
+```
+
+Additionally, add reversed Client ID com.googleusercontent.apps.{ios-client.id}
+as iOS URL scheme to Info.plist.\
```

Comment:
Is this what you need to add, or does this need to be reversed? Could you add what you need to the `Info.plist` file?